gaudi/README.md

```diff
@@ -0,0 +1,84 @@
+# Setting up HabanaAI Operator
+
+## Overview
+The steps mentioned in this document to deploy and configure [Habana AI Operator](https://catalog.redhat.com/software/container-stacks/detail/64342b3bcbfbb9a6588ce8dd) on Red Hat OpenShift are consolidated and referenced from [HabanaAI Operator for OpenShift](https://docs.habana.ai/en/latest/Orchestration/HabanaAI_Operator/index.html).
+
+## Prerequisities
+- Provisioned RHOCP cluster. Follow steps [here](/README.md).
+
+## Setting up OpenShift Environment
+- Setup Machine Configuration to update Firmware search path using below command. Please note that this step will reboot the nodes.
+```
```

Comment:
Can we change it
## Update Kernel firmware search path with MCO
Notes: This step will reboot the nodes, it is recommended to do this in the first step.  

The default Kernel firmware search path /lib/firmware in RHCOS is not writable. Command below can be used to add path /var/lib/fimware into the firmware search path list.

Reply:
Made the requested changes.